bpf/bpf_lxc.c

```diff
@@ -961,9 +966,14 @@ __section_tail(CILIUM_MAP_POLICY, LXC_ID) int handle_policy(struct __sk_buff *sk
 	}
 #endif
 
-	if (IS_ERR(ret))
+    //ingress
+	if (IS_ERR(ret)) { // dropped
 		return send_drop_notify(skb, src_label, SECLABEL, LXC_ID,
-					ifindex, ret, TC_ACT_SHOT);
+					ifindex, ret, TC_ACT_SHOT, DIRECTION_INGRESS);
+	} else { // forwarded
```

Comment:
The else is not required as you return above

Reply:
by bad. will fix it

Reply:
done